src/swarm/util/log/ClientStats.d

```diff
@@ -146,6 +152,40 @@ public class ClientStats : StatsLog
     }
 
 
+    /***************************************************************************
+
+        Construct the class to use the default StatsLog appender, registers it
+        with the applications' reopenable files extension (if present), and
+        starts the update timer.
+
+        Params:
+            app = application class
```

Comment:
Interesting that you chose to pass the application instance. Why not just the reopenable files extension?

Reply:
The idea was that it's slightly more convenient for the user this way.